_opensearch/cluster.md

```diff
@@ -226,6 +230,28 @@ If that is not the case, and `opensearch-d1` and `opensearch-d2` do not have the
 
 Choosing allocation awareness or forced awareness depends on how much space you might need in each zone to balance your primary and replica shards.
 
+### Replica count enforcement
+
+To enforce an even distribution of shards across all zones and avoid hotspots, you can set the `routing.allocation.awareness.balance` attribute to `true`. This setting can be configured in the opensearch.yml file and dynamically updated using the cluster update settings API:
+
+```json
+PUT _cluster/settings
+{
+  "persistent": {
+    "cluster": {
+      "routing.allocation.awareness.balance": "true"
+    }
+  }
+}
+```
+
+The `routing.allocation.awareness.balance` setting is false by default. When it is set to `true`, the total number of shards for the index must be a multiple of the highest count for any awareness attribute. For example, consider a configuration with two awareness attributes set up — zones and rack IDs. Let's say there are two zones and three rack IDs. The highest count of either the number of zones or the number of rack IDs is three. Therefore, the number of shards must be a multiple of three. If it is not, OpenSearch throws a validation exception.
```

Comment:
In the third sentence, can we remove "set up"? It looks like the sentence would work without it. Change the en dash to an em dash with no spaces on either side ("attributes—zones and rack IDs.").